ansible/roles/ocp4-workload-quarkus-workshop/files/stack.Dockerfile

```diff
@@ -1,13 +1,13 @@
 # To build this stack:
 # docker build -t quay.io/username/quarkus-workshop-stack:VVV -f stack.Dockerfile .
-# docker push quay.io/username/quarkus-workshop-stack:VVVV
+# docker push quay.io/username/quarkus-workshop-stack:VVVV 
```

Comment:
Added dangling whitespace?

Reply:
It's for creating a new PR without any changes.

Reply:
removed the white sapce.